observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -24,6 +24,45 @@ The following table provides information about which features are available depe
 | 1.1+          | 4.18+              | ✔                     | ✔         | ✔                     | ✔          | ✔
 |===
 
+[id="cluster-observability-operator-release-notes-1-2_{context}"]
+== {coo-full} 1.2
+
+// Back fill advisory when it is created by Konflux
+// The following advisory is available for {coo-full} 1.2:
+//
+// * link:https://access.redhat.com/errata/RHSA-2025:????[RHEA-2025:??? {coo-full} 1.2]
+
+[id="cluster-observability-operator-1-2-new-features-enhancements_{context}"]
+=== New features and enhancements
+
+* The logging UI plugin now supports the OTEL format, in addition to the previously supported ViaQ scheme. (link:https://issues.redhat.com/browse/COO-816[*COO-816*])
+
+* Accelerators Perses dashboards are deployed by default when you install the monitoring UI plugin. (link:https://issues.redhat.com/browse/COO-942[*COO-942*]) 
```

Comment:
Accelerators Perses dashboards are deployed by default if you enable  Perses when create the monitoring UI plugin.
I suppose better change as the above, please help confirm 
@etmurasaki

Reply:
Hi team, Accelerators Perses dashboards are deployed by default when you install Monitoring UIPlugin, independently of enabling Perses flag. 
It was a discussion and agreement between @jgbernalp and @jan--f with @yevgeny-shnaidman.
We had some fixes during COO release testing to make it works like this.